User-Help/SearchHelp.md

```diff
@@ -0,0 +1,59 @@
+## Codidact search options
+
+While searching on a Codidact site, you have several options to help refine your search and make it easier to find what you're looking for. Searching will search all posts across that site, including Questions, Answers, Meta posts, and whatever other post types your site may have.
+
+**Basic keyword searching**
+
+Let's say you want to search for the term `snake oil`. You have a few basic options:
+
+* plain search
+
+  If you put the text `snake oil` into the searchbox, Codidact will find all posts that contain *either* the word `snake` or `oil`.
+
+* exact search (quote marks)
+
+  If you include quote marks around your search term, such as `"snake oil"`, Codidact will find all posts that contain exactly the phrase `snake oil`.
+
+* keyword search (plus symbol)
+
+  If you search for `+snake +oil`, Codidact will find all posts that contain both the words `snake` and `oil`.
+
+* excluding words (minus symbol)
+
+  If you want to search for `snake` but exclude the word `oil` from your results, you can do a search for `snake -oil`. Codidact will find posts that contain the word `snake` but do not have the word `oil`.
+
+**Filtering by score and age**
+
+It's possible to filter your search to only include results that have been posted within a certain timeframe, or match certain score requirements.
+
+* filtering by Wilson score
```

Comment:
Let's just call this score, otherwise somebody skimming for score or net votes might think this is something more esoteric.  Wilson scoring *is* our scoring system; people from SE expect a different system and we'll have to teach them, but we shouldn't qualify the term on our own site.  (I'm talking here about the heading; in the paragraph that follows it's fine to say it, because that's just explaining how we do scores.)